woocommerce/woocommerce
# Add/34 create new category field modal

### All Submissions:






### Changes proposed in this Pull Request:

Partialy closes https://github.com/woocommerce/woocommerce/issues/35173

Adds a create category option to the categories dropdown when the search value does not exist.

https://user-images.githubusercontent.com/2240960/197212966-a646a474-1c14-4728-932a-5b07f59b6a9d.mp4


### How to test the changes in this Pull Request:

1. Load this branch, build it, and make sure you have the `new-product-management-experience` feature flag enabled (you can do so using the latest version of the Beta tester within the mono repo).
2. Go to **Products > Add New (MVP)**
3. Start typing in the search value of the Categories dropdown
4. If the search value you typed does not match an exact category a `+ Create 'category name'` option should show
5. Click the create option and a Modal should show with the category name pre-filled with the search value.
6. You have the option to select a parent (this is a single select dropdown in a similar format as the category dropdown)
7. Once you click **Save** it should create the category and add it as a selected value to the Categories list.

### Other information:




### FOR PR REVIEWER ONLY:
